src/mokaQtIhm/windows/gl-window.qt.cc

```diff
@@ -127,7 +127,7 @@ void GLWindow::closeEvent(QCloseEvent * e)
 //************************************************************
 void GLWindow::paintGL()
 {
-  QPainter p(this);
+//  QPainter p(this);
```

Comment:
Instead of commenting all these code, it is better to protect it by #ifndef directives, so that by default this code is here, and we can remove it by defining an option in CMake.
About naming, I suggest DISABLE_SELECTION_BOX (but feel free to suggest any better name if you have some idea).